TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2017/3l_semiddbkg/step3/mca-3l-applmc3fakesSF3F.txt

```diff
@@ -1,40 +1,1042 @@
-Fakes_tt       : TTJets_DiLepton                   : ScaleFactor3F*xsec : LepGood1_mcMatchId==0 || LepGood2_mcMatchId==0 || LepGood3_mcMtachId==0 ; FakeRate="susy-sos-v2-clean/fakerate/2017/3l/fakerate_applmc3.txt, FillColor=ROOT.kGray, Label="tt(fakes)"
```

Comment:
I think something went wrong here with the wget command. Can you please check and re-commit?